src/__tests__/tv.test.ts

```diff
@@ -2628,3 +2628,244 @@ describe("Tailwind Variants (TV) - Tailwind Merge", () => {
     expect(result).toHaveClass(["text-medium", "text-blue-500", "w-unit-4"]);
   });
 });
+
+describe("Tailwind Variants (TV) - Required Variants", () => {
```

Comment:
please add a test case for invalid variant names in requiredVariants.

Reply:
added.